### ISSUE TITLE
PR #350 probably broke linting

### ISSUE THREAD

**Issue by martenson** [galaxyproject/planemo#354]
dying with exception here
https://travis-ci.org/galaxyproject/tools-devteam/builds/87752155

ping @erasche 

```
File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/planemo_ext/galaxy/tools/lint.py", line 43, in lint
    lint_func(lint_target, self)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/planemo_ext/galaxy/tools/linters/inputs.py", line 34, in lint_inputs
    if not len(select) and not len(boolean):
TypeError: object of type 'NoneType' has no len()
```

**Comment by jmchilton** [comment]
Travis reruns verify this is fixed with b054cbecc82b37dc212060f3b35332f68a490787.

**Comment by erasche** [comment]
Sigh...OK. Thanks for finding this. :-)
